rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -40,86 +41,132 @@ def imagesGenJpgs(self, **kwargs: dict) -> None:
         jpgWidth = str(kwargs.get("jpgWidth"))
         jpgFormat = kwargs.get("jpgFormat")
         checkFileAppend = kwargs.get("checkFileAppend", "_model-1.jpeg")
-        pdbBaseDir = kwargs.get("pdbBaseDir")
-        csmBaseDir = kwargs.get("csmBaseDir")
+        baseDir = kwargs.get("baseDir")
         jpgsOutDir = kwargs.get("jpgsOutDir")
-        contentTypeDir = kwargs.get("contentTypeDir")
         numProcs = kwargs.get("numProcs")
 
-        logger.info("using id file %s", idListFile)
+        holdingsFilePath = Path(kwargs.get("holdingsFilePath"))
+        targetFileSuffix = kwargs.get("targetFileSuffix")
 
+        # load id list file
+        logger.info("using id file %s", idListFile)
         listFileObj = Path(idListFile)
         if not (listFileObj.is_file() and listFileObj.stat().st_size > 0):
-            logger.error("Missing idList file %s", idListFile)
-            return
-
+            raise RuntimeError(f"Missing idList file {idListFile}")
         mU = MarshalUtil()
         idList = mU.doImport(idListFile, fmt="list")
         if not isinstance(idList, list) and not idList:
             raise TypeError("idList not a list or is empty.")
 
+        # get holdings file dict for their timestamps
+        if "computed-models-holdings-list" in str(holdingsFilePath):
```

Comment:
Relying on a very specific (and quite long) string seems dangerous (it is easy to make mistakes, it is not very transparent to users of the CLI tool). Could this be done differently? Passing a flag perhaps?